LoopUI/Views/LoopStatusCircleView.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  LoopStatusCircleView.swift
+//  LoopUI
+//
+//  Created by Arwain Karlin on 5/8/24.
+//  Copyright © 2024 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+public struct LoopStatusCircleView: View {
+    
+    public enum Status {
+        case closedLoopOn
+        case closedLoopOff
+        case closedLoopUnavailable
+        
+        var color: Color {
+            switch self {
+            case .closedLoopOn:
+                return .green  // Use guidanceColors
+            case .closedLoopOff:
+                return .red  // Use guidanceColors
+            case .closedLoopUnavailable:
+                return .orange // Use guidanceColors
+            }
+        }
+    }
+    
+    @Binding var closedLoop: Bool
+    @Binding var closedLoopAvailable: Bool
```

Comment:
Thinking we may want this to be `isClosedLoopAllowed` to better match up with the parameters passed in at the call-site.